frontend/src/views/DashboardHeader.vue

```diff
@@ -298,54 +227,6 @@ export default defineComponent({
       return inboxStore.getInboxSummaryByUser(currentUser.value.id);
     });
 
-    const switchToOwner = () => {
-      authStore.login({
-        authProvider: "BYTEBASE",
-        payload: {
-          email: "demo@example.com",
-          password: "1024",
-        },
-      });
-    };
-
-    const switchToDBA = () => {
-      authStore.login({
-        authProvider: "BYTEBASE",
-        payload: {
-          email: "jerry@example.com",
-          password: "2048",
-        },
-      });
-    };
-
-    const switchToDeveloper = () => {
-      authStore.login({
```

Comment:
If we encounter a problem using the quick switch, then we should look into why that happens.

But from the code point of view, we are following the standard workflow to send API to backend for switching roles.

We curated the seed data to achieve this https://github.com/bytebase/bytebase/blob/main/store/demo/dev/10001%23%23principal.sql#L22